### ISSUE TITLE
Migrate About page

### ISSUE THREAD

**Issue by AndrewJDick** [fathomlondon/fathomlondon.com#188]

**Comment by jfidiles** [comment]
I might start migrating this page once #190 is merged

**Comment by peduarte** [comment]
@jfidiles sounds good, let's have a catch up before hand 👍

**Comment by jfidiles** [comment]
Will do #185 instead

**Comment by AndrewJDick** [comment]
Closing in favour of https://github.com/fathomlondon/fathomlondon.com/pull/203

**Comment by artisologic** [comment]
Closed via #203
